src/tests/EdFi.DataManagementService.Tests.E2E/Features/DeleteReferenceValidation.feature.disabled

```diff
@@ -0,0 +1,93 @@
+Feature: Validation of DELETE requests that would cause a foreign key violation
+
+        Background:
+            Given this School Year Type exists
+                  | id                               | schoolYear | currentSchoolYear | schoolYearDescription |
+                  | 20906a639ed04dfa9db77107f2c7834a | 2022       | false             | 2021-2022             |
+              And this Education Organization Category Descriptor exists
+                  | id                               | codeValue              | namespace                                               | shortDescription       |
+                  | 8f2be92ff82641cc99ebff0d14c29176 | Local Education Agency | uri://ed-fi.org/EducationOrganizationCategoryDescriptor | Local Education Agency |
+              And this Student exists
+                  | id                               | studentUniqueId | birthDate  | firstName | lastSurname |
+                  | 64dc6d7916064379a5c0be1f6e5195a5 | 604824          | 2010-01-13 | Traci     | Mathews     |
+              And this Education Organization (Local Education Agency) exists
+                  | id                               | categories                 | localEducationAgencyId | localEducationAgencyCategoryDescriptor | nameOfInstitution |
+                  | d9d973856819497e81e6e18a4d8f721d | ["Local Education Agency"] | 255901                 | ["Independent"]                        | Grand Bend ISD    |
+              And this program exists
+                  | id                               | programName                    | programTypeDescriptor              | educationOrganizationId |
+                  | 4dbd101cde9942f09276ebe0628e5096 | Career and Technical Education | ["Career and Technical Education"] | 255901                  |
+
+        @ignore
+        Scenario: Ensure clients cannot delete a year that is used by another item
+             When a DELETE request is made to "ed-fi/schoolYearTypes/20906a639ed04dfa9db77107f2c7834a"
```

Comment:
I don't know how we're going to handle this. We don't know what the `id` value is, so we this line will never work. Since only one school year is setup in the background, maybe it would be okay to put `"ed-fi/schoolYearTypes/{id}"`. We would need a different approach if there were multiple school years, and we may need to refactor this again later.

Reply:
A possible alternative: `When trying to delete school year 2022`, but that step definition will never be reused anywhere.

Reply:
There is an alternative way to retrieve data